client/lib/media/actions.js

```diff
@@ -37,7 +37,11 @@ MediaActions.setQuery = function( siteId, query ) {
 	} );
 };
 
-MediaActions.fetch = function( siteId, itemId ) {
+// TODO: Once this entire module is converted to Redux,
+// `receiveMediaItems` shouldn't be passed as an argument anymore,
+// but directly imported and dispatch()ed from inside `fetch()`,
+// which needs to be turned into a Redux thunk.
+MediaActions.fetch = function( siteId, itemId, receiveMediaItems = null ) {
```

Comment:
I guess there's a point to that since #2167 looks small enough that we can hopefully rebase and merge it easily, and there's going to be a number of other occurrences where we'd mess with Flux action signatures in a similar fashion otherwise. So yeah, given our restrengthened Reduxification efforts, this might make sense.

Sorry for originally suggesting the other approach 😞